src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/CuratedContentToolModal.java

```diff
@@ -0,0 +1,30 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.modalwindows;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Created by wikia on 2015-08-19.
+ */
+public class curatedContentToolModal extends WikiBasePageObject {
```

Comment:
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).
~ http://www.oracle.com/technetwork/java/codeconventions-135099.html